packages/template/package.json

```diff
@@ -36,7 +36,7 @@
         "serve": "cross-env serve --single build",
         "test": "react-scripts test --env=jsdom",
         "eject": "react-scripts eject",
-        "interbit:start": "interbit start --db-path ./db-interbit --port 5000 --dev --no-watch",
+        "interbit:start": "interbit start --db-path ./db-interbit --port 5000 --dev",
         "interbit:create": "interbit create --dev",
```

Comment:
Can the --no-watch switch be passed in? In other words, is the following valid?
```
npm run interbit:start --no-watch
```
It should be!

Reply:
Yes it can! If you want to run it --no-watch you can type -- [youradditionalargs] to pass args to a lower command.

```
npm run interbit:start -- --no-watch
```